app/views/layouts/application.html.erb

```diff
@@ -101,18 +101,40 @@
   <%= render "govuk_publishing_components/components/layout_footer", {
     navigation: [
       {
-        title: "Support and feedback",
+        title: t("application.footer.support_and_feedback"),
         items: [
           {
-            href: "https://support.publishing.service.gov.uk/",
+            href: "https://signon.integration.publishing.service.gov.uk/users/sign_in",
             text: "Raise a support request"
           },
+          {
+            href: "https://signon.integration.publishing.service.gov.uk/users/sign_in",
```

Comment:
Same comment about Plek and sign in form

Reply:
FIXED.  Plek.new.external_url_for("support") works nicely

Reply:
In case you missed the Slack comment, these now seem to both link to the same root path of support. Is that intentional?

Reply:
Yes, until we actually make the new pages that can differentiate between support request and send us feedback.